.circleci/config.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - run: npm install
       - run: npm test
 
-  deploy:
+  release:
```

Comment:
I've renamed this to `release` because effectively we're not deploying anything.